data/mods/Magiclysm/items/ethereal_items.json

```diff
@@ -1238,5 +1238,24 @@
       "PSYSHIELD_PARTIAL",
       "PORTAL_PROOF"
     ]
+  },
+  {
+    "id": "druid_bandages",
+    "type": "GENERIC",
+    "category": "drugs",
+    "name": { "str": "Vegetative Poultice" },
+    "looks_like": "leaves",
+    "description": "A mixture of bark, moss, and leaves, glowing faintly with verdant energy.",
+    "weight": "70 g",
+    "volume": "250 ml",
+    "price": 1,
+    "price_postapoc": 1,
+    "material": [ "wood" ],
+    "symbol": "!",
+    "color": "green",
+    "charges": 3,
```

Comment:
Does this means it spawn 3 bandages at one cast?

Reply:
No, it only spawns one and each bandage only has one use. When I wrote this spell a couple months ago, every bandage had 3 charges (while still only having one use), so I assumed it was some weird legacy code thing I had to include. Bandages no longer have three charges, though, and testing shows it works fine without it, so I'll take it out.